newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -146,18 +167,18 @@ export default class PixiResourcesLoader {
     const resource = project.getResourcesManager().getResource(resourceName);
     if (resource.getKind() !== 'video') return invalidTexture;
 
-    loadedTextures[resourceName] = PIXI.Texture.from(
-      ResourcesLoader.getResourceFullUrl(project, resourceName, {
-        disableCacheBurst: true, // Disable cache bursting for video because it prevent the video to be recognized as such (for a local file)
-        isResourceForPixi: true,
-      }),
-      {
-        scaleMode: PIXI.SCALE_MODES.LINEAR,
-        resourceOptions: {
-          autoPlay: false,
-        },
-      }
-    );
+    const url = ResourcesLoader.getResourceFullUrl(project, resourceName, {
+      disableCacheBurst: true, // Disable cache bursting for video because it prevent the video to be recognized as such (for a local file)
```

Comment:
```suggestion
      disableCacheBurst: true, // Disable cache bursting for video because it prevents the video to be recognized as such (for a local file).
```